### PR TITLE
Rahul Fix Teams Modal UI

### PR DESCRIPTION
**Pull request by slidracoon72**
# Description
Screenshot 2024-10-29 at 12 42 40 AM
This PR addresses the UI bug in the Teams Modal where long usernames caused overflow issues and inconsistencies with the delete button and background, especially in dark mode. The update ensures a streamlined and visually consistent appearance for both dark and light modes, with the delete button properly aligned and a uniform background display.

## Related PRS (if any):
Not related to any PRS

## Main changes explained:
- Updated file "**TeamMembersPopup.jsx**" to improve the layout and styling of the Teams Popup Modal

## How to test:
1. check into current branch: `git checkout rahul-fix-teams-modal-ui`
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin/owner user
5. go to dashboard → Other Links → Teams → Select any team having members with long names
6. if you cannot find a team having members with long names, create a new user with a long name from User Management and add the new member to a team
7. verify that the modal's background is consistent is dark mode upon adding members with long names
8. verify that the delete button's background is not white and it is consistent with the modal in dark and light mode
9. checkout the below BEFORE and AFTER videos to get a better understanding of the changes made

## Videos of changes:
**BEFORE**

https://www.loom.com/share/00fc6912afe14ce28a57075fd75c1d34?sid=e9ca3f9b-4eda-4195-ab0b-08c9d846270a 

**AFTER**

https://www.loom.com/share/588d98a1faef4a3c8698512a9ac85cef?sid=eb049f05-3053-4b11-974d-9c7119fe5f59

### CODE REVIEW

**Review by samarth9201 — APPROVED** [review]
Verified the changes. LGTM!
Screenshot 2024-11-01 002416

**Review by anirudhsk2107 — APPROVED** [review]
The changes are working. Tested for light and dark mode both. 

Screen Shot 2024-11-01 at 12 42 58 PM
Screen Shot 2024-11-01 at 12 41 41 PM

**Review by hqzhu0913 — APPROVED** [review]
No UI problem in the dark mode in both desktop and mobile.
Screenshot 2024-11-01 at 3 03 47 PM
Screenshot 2024-11-01 at 3 06 05 PM

**Review by SupriyaSudini — DISMISSED** [review]
I have reviewed the PR and confirmed that the delete button displays correctly with a good background in dark mode, and the functionality is working as expected. Approved.

image

image

image

**Review by Rui-Liu-github — APPROVED** [review]
Hi， I reviewed PR #2831 and tested the teams model UI. It works well in both dark mode and light mode.

image
image
image
image

**Review by vijay10pk — DISMISSED** [review]
All the changes are working as expected in light/dark mode and mobile screen.
Screenshot 2024-11-01 at 10 13 11 PM
Screenshot 2024-11-01 at 10 12 49 PM
image

**Review by manikittu810 — DISMISSED** [review]
Hello @slidracoon72 
Everything is working good.
Screenshot 2024-11-01 at 7 48 55 PM
Thanks

**Review by Jingyii800 — APPROVED** [review]
I have tested this feature. Now the team modal looks good even if the member has a very long name. And it works well in both light and dark mode.

https://github.com/user-attachments/assets/a39e8d1a-1faf-45cc-a5bd-6ca32a951850

**Review by anji-annavarapu — APPROVED** [review]
Screenshot 2024-11-02 010127

view delete button now normal

**Review by nikhilgiri0226 — DISMISSED** [review]
I have reviewed this PR and it's working as expected. The delete button UI was consistent in both light and dark modes.

image

image

image

**Review by varun-elango — APPROVED** [review]
The functionality is working as expected. The modal background is consistent in light and dark modes for teams with members having long names.
PR2831_1
PR2832_2

File: src/components/Projects/WBS/WBSDetail/wbs.css [hunk 1]
```diff
@@ -235,7 +235,7 @@
 }
 .tasks-detail-task-name { /* task column height and width changes*/
   width: 35%; 
-.tasks-detail-task-name { /* task column height and width changes*/
+ /* task column height and width changes*/
```
[the remark this answers is not in the document]
Comment (slidracoon72) [reply src/components/Projects/WBS/WBSDetail/wbs.css]: @sheetalmangate I have removed the comment.

**Review by sheetalmangate — DISMISSED** [review]
Tested this PR locally. its working as per requirement so approving this PR.

PR2831_testing

**Review by aaryaneil — APPROVED** [review]
Tested the PR and the UI doesn't have any issue in case of long names of team member.
Screenshot 2024-11-02 at 9 15 35 PM

**Review by humera314 — APPROVED** [review]
It works as expected
image

**Comment by SatyashanthiT** [comment]
I tested this PR and confirmed that the Teams Modal UI is working as expected. The layout and styling improvements for both dark and light modes are consistent, and the delete button alignment issue is resolved. 
image
image

**Comment by RashmiKY98** [comment]
Tested in dark mode and works fine. Observed the changes by varying the webpage height and width. No issues found. Tested in light mode as well.
Screenshot 2024-11-02 223343
Screenshot 2024-11-02 223153

**Comment by geetamatkar** [comment]
Tested this PR and the functionality is working as expected. The modal background is consistent in light and dark modes for teams with members having long names.

Screenshot 2024-11-08 at 6 54 14 PM
Screenshot 2024-11-08 at 6 54 42 PM

**Review by anandsesha — APPROVED** [review]
Tested this PR and the text overflow issues and inconsistencies with the delete button are fixed. This update is visually consistent for both dark and light modes, with the delete button properly aligned and a uniform background display. screenshots attached.

Screenshot (1124)
Screenshot (1123)
Screenshot (1122)
Screenshot (1121)

**Review by SammanBaidya — DISMISSED** [review]
I have reviewed and tested the changes, and everything is working as expected. I tested both light and dark modes, and the issue has been well resolved.

PR3 1 #2831
PR3 2 #2831

**Comment by one-community** [comment]
Thank you all, merging!

**Pull request merged**